general_superstaq/finance_test.py

```diff
@@ -77,7 +80,9 @@ def test_service_submit_qubo(mock_submit_qubo: mock.MagicMock) -> None:
         "qubo": [{"keys": ["0"], "value": 123}],
     },
 )
-def test_service_find_min_vol_portfolio(mock_find_min_vol_portfolio: mock.MagicMock) -> None:
+def test_service_find_min_vol_portfolio(
+    mock_find_min_vol_portfolio: mock.MagicMock,
+) -> None:
```

Comment:
this change shouldn't be necessary i think
```suggestion
def test_service_find_min_vol_portfolio(mock_find_min_vol_portfolio: mock.MagicMock) -> None:
```